library/pod-security-policy/apparmor/src_test.rego

```diff
@@ -0,0 +1,62 @@
+package k8spspapparmor
+
+test_input_apparmor_allowed_in_list {
+    input := { "review": input_review, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_apparmor_not_allowed_not_in_list {
+    input := { "review": input_review, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) > 0
+}
+
+test_input_apparmor_not_allowed_no_annotation {
+    input := { "review": input_review_no_annotation, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) > 0
```

Comment:
Can you pls link to the doc where it says `apparmor.security.beta.kubernetes.io/pod` must be provided?

Reply:
looks like AppArmor is only on container level, removed pod level for it
https://kubernetes.io/docs/tutorials/clusters/apparmor/#securing-a-pod